poetry.lock

```diff
@@ -1,655 +1,231 @@
-[extras]
-dev = ["pytest", "flake8", "coverage", "mypy", "tox"]
+[[package]]
+name = "atomicwrites"
+version = "1.4.1"
+description = "Atomic file writes."
+category = "main"
+optional = false
+python-versions = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*"
```

Comment:
Just following up on your comment on the PR description. With poetry, it's good to commit changes to the `poetry.lock` file so all developers can be working off the same supporting packages. One thing I noticed though is it looks like `pre-commit` didn't get a chance to reformat the LockFile so the diff looks quite big. Would you mind running pre-commit on the code base one time for some formatting? 

https://juftin.com/camply/contributing.html#coding

Since the commits are already in you can simply run it for all files
```shell
pipx install pre-commit
pre-commit install
pre-commit run --all-files
```

Reply:
Wow! Somehow I completely missed https://juftin.com/camply/ as another source of documentation! I wish I had seen this before. I've been semi-reverse-engineering how to run and test it myself.

I'll look through that and clean up the lock file.